### PR TITLE
Clarify login step for MCG

### PR DESCRIPTION
**Pull request by apinnick**
Added OpenShift to the login step for creating MCG CRs. 
DDF bug: https://bugzilla.redhat.com/show_bug.cgi?id=1828695
CP for 4.2, 4.3, 4.4.

### CODE REVIEW

File: modules/migration-configuring-mcg.adoc [hunk 1]
```diff
@@ -6,12 +6,12 @@
 [id='migration-configuring-mcg_{context}']
 = Configuring a Multi-Cloud Object Gateway storage bucket as a replication repository
 
-You can install the OpenShift Container Storage Operator and configure a Multi-Cloud Object Gateway (MCG) storage bucket as a replication repository on the CAM tool cluster.
+You can install the {product-title} Container Storage Operator and configure a Multi-Cloud Object Gateway (MCG) storage bucket as a replication repository.
```
Comment (bergerhoffer) [inline modules/migration-configuring-mcg.adoc:9]: I don't think that all these changes from "OpenShift Container Storage Operator" to "product-title} Container Storage Operator" are correct. Now it says "OpenShift Container Platform Container Storage Operator". And if you're talking about the product "OpenShift Container Storage" [1], then you had it right the first time.

[1] https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.2/html-single/deploying_openshift_container_storage/index#installing-openshift-container-storage-operator-using-the-operator-hub_rhocs

**Review by bergerhoffer — COMMENTED** [review]
Just one thing questioning the changes to {product-title}

File: modules/migration-configuring-mcg.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-configuring-mcg.adoc:9]: Thanks! Will fix

**Pull request merged**